### DISCUSSION TITLE
Different Dependency Location

### DISCUSSION THREAD

**Discussion by ProfessorManhattan** [chrissimpkins/Crunch#105] [General]
Is there anyway to install this in such a way that `~/zopfli` and `~/pngquant` don't pollute the HOME directory? (i.e. can we specify custom locations?)

Also, it'd be nice if a Homebrew formulae was published for the CLI

**Comment by chrissimpkins** [comment]
It's been some time since I last worked on the sources here, but it looks like I hard coded all of the paths in variables that are in the head section of the python file here:

https://github.com/chrissimpkins/Crunch/blob/a343532e128bef8d49636b7044b012bf05824658/src/crunch.py#L41-L48

Just modify those paths and install wherever makes most sense to you.

**Reply by chrissimpkins** [reply]
The dependency installation script can be updated with the same paths that you choose:

https://github.com/chrissimpkins/Crunch/blob/master/src/install-dependencies.sh
